pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -120,12 +119,15 @@ func NewService(srv *scheserver.Service) *Service {
 func (s *Service) RegisterSchedulersRouter() {
 	router := s.root.Group("schedulers")
 	router.GET("", getSchedulers)
+	router.GET("/diagnostic/:name", getDiagnosticResult)
+	router.POST("/:name", pauseOrResumeScheduler)
```

Comment:
Shall we split it into two APIs?

Reply:
No, we only support one-to-one forwarding at this time.

Reply:
What I mean is since we use the new framework for scheduling service, we should make it more standard. The previous one is not good enough, so we should make it more clear.

Reply:
If we want to achieve one-to-two forwarding, i.e. post `scheduler/{name}` is at the api server and forwards to post `scheduler/{name}/pause` and post `scheduler/{name}/resume`, we may need to add a check for body to the match rule, which I'll try to add.

Reply:
If we want to achieve one-to-two forwarding, i.e. post `scheduler/{name}` is at the api server and forwards to post `scheduler/{name}/pause` and post `scheduler/{name}/resume`, we may need to add a check for body to the match rule.

Reply:
We should make the new API more clear.

Reply:
If you don't want to change it and since it is only used by forwarding now, I'm ok with the current status. But in the future, it's better to separate these actions.

Reply:
ok, I will add TODO about it.